cmd/autoowners/main.go

```diff
@@ -302,10 +302,14 @@ func pullOwners(directory string, whitelist []string) ([]string, error) {
 		}
 
 		err = orgRepo.getOwnersHTTP()
-		if err != nil && !os.IsNotExist(err) {
+		if err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Huh how does this work? To me it looks like that when upstream has neither `OWNERS` nor `OWNERS_ALIASES` then `getOwnersHTTP()` returns `nil`, not an error (https://github.com/openshift/ci-tools/blob/78701e4959c9b8fc64edc64d55f00dcdc01113d8/cmd/autoowners/main.go#L161)... The code there seems to handle the FileNotFound case https://github.com/openshift/ci-tools/blob/78701e4959c9b8fc64edc64d55f00dcdc01113d8/cmd/autoowners/main.go#L135 and just continue...

Reply:
Did not know what I was thinking ... Let me test it locally.

Reply:
/hold
found more issues

Reply:
/hold cancel

Tested with https://github.com/openshift/release/pull/5025